src/test/java/hw2/ex1/FirstScenarioTest.java

```diff
@@ -0,0 +1,203 @@
+package hw2.ex1;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import org.testng.annotations.AfterTest;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class FirstScenarioTest {
+
+    private WebDriver driver;
+
+    @BeforeTest
+    public void setUp() {
+        WebDriverManager.chromedriver().setup();
+        this.driver = new ChromeDriver();
+        driver.manage().timeouts().
+                implicitlyWait(10000, TimeUnit.MILLISECONDS);
+        driver.manage().timeouts().
+                pageLoadTimeout(20000, TimeUnit.MILLISECONDS);
+        driver.manage().timeouts().
+                setScriptTimeout(30000, TimeUnit.MILLISECONDS);
```

Comment:
Browser window does not maximized as described in the requirements

Reply:
Well, I fixed it